### ISSUE TITLE
Use several apps

### ISSUE THREAD

**Issue by jeremygachet** [richsage/RMSPushNotificationsBundle#102]
Hello, 
I'm working on a project with 2 applications. 
I would like to set 2 possibles pushing services, and choose wich one to use when i'm creating mu notification. 
Is it possible with your bundle and how can we do this ?

**Comment by richsage** [comment]
Could you be a bit more specific regarding "2 possible pushing services"?

**Comment by jeremygachet** [comment]
I have an IOS application for clients, and another IOS Application for some messengers. (with 2 different .pem)

Depending on the events in the app, i would like to send notifications to users of application 1  or to users on application 2

I imagine something like: 
$message = new iOSMessage("nameOfTheAppInConfig");
$message->setMessage($msg);

**Comment by peternijssen** [comment]
Same here. Also working with multiple apps and would be nice if all could be configured through this bundle :)

**Comment by richsage** [comment]
@peternijssen @jeremygachet yes, I think #76 will be the answer to this. @jeremygachet could you take a look and see if that will solve your issue?

**Comment by jeremygachet** [comment]
Hello and thank you for your answer.
Unfortunatly I had to choose another solution (I user another library for the second app)
But I think that it's exactly what I needed.
My next dev with your bundle will be more clean. 
Thanks a lot!
(If someone could update the doc it would be absolutly perfect)
